HTTP/1.1 200 OK
Transfer-Encoding: chunked
Access-Control-Allow-Origin: *
Access-Control-Expose-Headers: ETag, Link, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval
Cache-Control: no-cache
Content-Security-Policy: default-src 'none'
Content-Type: application/json; charset=utf-8
Date: Sat, 09 Jun 2018 08:11:17 GMT
Link: <https://api.github.com/search/issues?q=repo%3Abcit-ci%2FCodeIgniter+type%3Aissue+state%3Aclosed&page=2>; rel="next", <https://api.github.com/search/issues?q=repo%3Abcit-ci%2FCodeIgniter+type%3Aissue+state%3Aclosed&page=34>; rel="last"
Referrer-Policy: origin-when-cross-origin, strict-origin-when-cross-origin
Server: GitHub.com
Status: 200 OK
Strict-Transport-Security: max-age=31536000; includeSubdomains; preload
X-Accepted-Oauth-Scopes: 
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-Github-Media-Type: github.v3; format=json
X-Github-Request-Id: CE67:57C5:373382F:71A8F0A:5B1B8BA4
X-Oauth-Scopes: 
X-Ratelimit-Limit: 30
X-Ratelimit-Remaining: 10
X-Ratelimit-Reset: 1528531917
X-Runtime-Rack: 0.405992
X-Xss-Protection: 1; mode=block

8000
{"total_count":2800,"incomplete_results":false,"items":[{"url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5522","repository_url":"https://api.github.com/repos/bcit-ci/CodeIgniter","labels_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5522/labels{/name}","comments_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5522/comments","events_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5522/events","html_url":"https://github.com/bcit-ci/CodeIgniter/issues/5522","id":328744036,"node_id":"MDU6SXNzdWUzMjg3NDQwMzY=","number":5522,"title":"url problem","user":{"login":"AlokInsan","id":39857336,"node_id":"MDQ6VXNlcjM5ODU3MzM2","avatar_url":"https://avatars0.githubusercontent.com/u/39857336?v=4","gravatar_id":"","url":"https://api.github.com/users/AlokInsan","html_url":"https://github.com/AlokInsan","followers_url":"https://api.github.com/users/AlokInsan/followers","following_url":"https://api.github.com/users/AlokInsan/following{/other_user}","gists_url":"https://api.github.com/users/AlokInsan/gists{/gist_id}","starred_url":"https://api.github.com/users/AlokInsan/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/AlokInsan/subscriptions","organizations_url":"https://api.github.com/users/AlokInsan/orgs","repos_url":"https://api.github.com/users/AlokInsan/repos","events_url":"https://api.github.com/users/AlokInsan/events{/privacy}","received_events_url":"https://api.github.com/users/AlokInsan/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":2,"created_at":"2018-06-02T13:25:10Z","updated_at":"2018-06-03T10:05:47Z","closed_at":"2018-06-02T14:00:11Z","author_association":"NONE","body":"I got an error, when i remove index.php by .htaacess file and replace from config folder $config['index_page'] = '';  also, and i am using ion auth for login purpose . I hit localhost/CI url i got Object not found message. and when i write index.php into config folder $config['index_page'] = 'index.php'; after i hit url it works fine. the url is made like this localhost/CI/index.php/auth/login.\r\ni want to remove index.php completely from url. my codeigniter version is CI 3.1.8\r\n\r\nso please help me any one.","score":1.0},{"url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5521","repository_url":"https://api.github.com/repos/bcit-ci/CodeIgniter","labels_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5521/labels{/name}","comments_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5521/comments","events_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5521/events","html_url":"https://github.com/bcit-ci/CodeIgniter/issues/5521","id":328734988,"node_id":"MDU6SXNzdWUzMjg3MzQ5ODg=","number":5521,"title":"Error Occur when I join table and gives duplicate value","user":{"login":"prashantrijal96","id":36468577,"node_id":"MDQ6VXNlcjM2NDY4NTc3","avatar_url":"https://avatars1.githubusercontent.com/u/36468577?v=4","gravatar_id":"","url":"https://api.github.com/users/prashantrijal96","html_url":"https://github.com/prashantrijal96","followers_url":"https://api.github.com/users/prashantrijal96/followers","following_url":"https://api.github.com/users/prashantrijal96/following{/other_user}","gists_url":"https://api.github.com/users/prashantrijal96/gists{/gist_id}","starred_url":"https://api.github.com/users/prashantrijal96/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/prashantrijal96/subscriptions","organizations_url":"https://api.github.com/users/prashantrijal96/orgs","repos_url":"https://api.github.com/users/prashantrijal96/repos","events_url":"https://api.github.com/users/prashantrijal96/events{/privacy}","received_events_url":"https://api.github.com/users/prashantrijal96/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":1,"created_at":"2018-06-02T11:02:44Z","updated_at":"2018-06-02T14:01:54Z","closed_at":"2018-06-02T14:01:54Z","author_association":"NONE","body":"I'm writing an application on CodeIgniter 3.1.6 and when I join 4 tables with the table \"order\", I got more than 1 output. I have only 2 rows on my table but the output in view are 6 rows, So, how can I solve this problem. ","score":1.0},{"url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5518","repository_url":"https://api.github.com/repos/bcit-ci/CodeIgniter","labels_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5518/labels{/name}","comments_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5518/comments","events_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5518/events","html_url":"https://github.com/bcit-ci/CodeIgniter/issues/5518","id":328125628,"node_id":"MDU6SXNzdWUzMjgxMjU2Mjg=","number":5518,"title":"Transaction status FALSE but DB error NULL","user":{"login":"csotelo8885","id":8917532,"node_id":"MDQ6VXNlcjg5MTc1MzI=","avatar_url":"https://avatars3.githubusercontent.com/u/8917532?v=4","gravatar_id":"","url":"https://api.github.com/users/csotelo8885","html_url":"https://github.com/csotelo8885","followers_url":"https://api.github.com/users/csotelo8885/followers","following_url":"https://api.github.com/users/csotelo8885/following{/other_user}","gists_url":"https://api.github.com/users/csotelo8885/gists{/gist_id}","starred_url":"https://api.github.com/users/csotelo8885/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/csotelo8885/subscriptions","organizations_url":"https://api.github.com/users/csotelo8885/orgs","repos_url":"https://api.github.com/users/csotelo8885/repos","events_url":"https://api.github.com/users/csotelo8885/events{/privacy}","received_events_url":"https://api.github.com/users/csotelo8885/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":2,"created_at":"2018-05-31T12:51:06Z","updated_at":"2018-06-01T14:56:02Z","closed_at":"2018-06-01T14:56:02Z","author_association":"NONE","body":"I'd like to catch the sqlsrv(driver) db-error from a codeigniter transac with insert_batch:\r\n\r\n```\r\npublic function sync($data)\r\n   {\r\n       $this->myDB->trans_start();\r\n           $this->myDB->insert_batch('myTable', $data);\r\n           $db_error = $this->myDB->error();\r\n       $this->myDB->trans_complete();\r\n\r\n       if ($this->myDB->trans_status() === FALSE)\r\n       {        \r\n           if (!empty($db_error['message']))\r\n           {\r\n               return $db_error['message'];\r\n           }\r\n       }\r\n       else\r\n           return true;\r\n   }\r\n}\r\n```\r\n\r\nThe trans_status() works fine because a generated SQL \"string truncated\" error. The problem is, accord with the CI3 system sqlsrv driver :\r\n\r\n```\r\npublic function error()\r\n   {\r\n       $error = array('code' => '00000', 'message' => '');\r\n       $sqlsrv_errors = sqlsrv_errors(SQLSRV_ERR_ERRORS);\r\n\r\n       if ( ! is_array($sqlsrv_errors))\r\n       {\r\n           return $error;\r\n       }\r\n...\r\n```\r\nSo the sqlsrv_errors(SQLSRV_ERR_ERRORS) returns NULL (Error number: 00000)\r\n\r\nBut if I set the CI config/database.php to 'db_debug' => TRUE, CI show that error:\r\n\r\n![capture](https://user-images.githubusercontent.com/8917532/40783184-4edb8faa-64a7-11e8-87ae-5c46affeed2a.JPG)\r\n\r\nWhy??, exists another way to capture the DB-error in a batch?\r\n\r\nThanks in advance,","score":1.0},{"url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5517","repository_url":"https://api.github.com/repos/bcit-ci/CodeIgniter","labels_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5517/labels{/name}","comments_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5517/comments","events_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5517/events","html_url":"https://github.com/bcit-ci/CodeIgniter/issues/5517","id":327932959,"node_id":"MDU6SXNzdWUzMjc5MzI5NTk=","number":5517,"title":"log files","user":{"login":"malohtie","id":20033279,"node_id":"MDQ6VXNlcjIwMDMzMjc5","avatar_url":"https://avatars1.githubusercontent.com/u/20033279?v=4","gravatar_id":"","url":"https://api.github.com/users/malohtie","html_url":"https://github.com/malohtie","followers_url":"https://api.github.com/users/malohtie/followers","following_url":"https://api.github.com/users/malohtie/following{/other_user}","gists_url":"https://api.github.com/users/malohtie/gists{/gist_id}","starred_url":"https://api.github.com/users/malohtie/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/malohtie/subscriptions","organizations_url":"https://api.github.com/users/malohtie/orgs","repos_url":"https://api.github.com/users/malohtie/repos","events_url":"https://api.github.com/users/malohtie/events{/privacy}","received_events_url":"https://api.github.com/users/malohtie/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":2,"created_at":"2018-05-30T23:21:15Z","updated_at":"2018-05-31T00:06:18Z","closed_at":"2018-05-30T23:42:35Z","author_association":"NONE","body":"i don't know if it a bug or not but my logs files jumped from log-2018-05-29.php to log-2018-05-31.php\r\n\r\n\r\n![image](https://user-images.githubusercontent.com/20033279/40752658-084cd5b8-6460-11e8-885d-92f0ff714817.png)\r\n","score":1.0},{"url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5515","repository_url":"https://api.github.com/repos/bcit-ci/CodeIgniter","labels_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5515/labels{/name}","comments_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5515/comments","events_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5515/events","html_url":"https://github.com/bcit-ci/CodeIgniter/issues/5515","id":327495911,"node_id":"MDU6SXNzdWUzMjc0OTU5MTE=","number":5515,"title":"codeigniter special characters of the type Ã©ðŸ”¥ ðŸ”¥","user":{"login":"spencer237","id":33651081,"node_id":"MDQ6VXNlcjMzNjUxMDgx","avatar_url":"https://avatars2.githubusercontent.com/u/33651081?v=4","gravatar_id":"","url":"https://api.github.com/users/spencer237","html_url":"https://github.com/spencer237","followers_url":"https://api.github.com/users/spencer237/followers","following_url":"https://api.github.com/users/spencer237/following{/other_user}","gists_url":"https://api.github.com/users/spencer237/gists{/gist_id}","starred_url":"https://api.github.com/users/spencer237/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/spencer237/subscriptions","organizations_url":"https://api.github.com/users/spencer237/orgs","repos_url":"https://api.github.com/users/spencer237/repos","events_url":"https://api.github.com/users/spencer237/events{/privacy}","received_events_url":"https://api.github.com/users/spencer237/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":1,"created_at":"2018-05-29T21:34:33Z","updated_at":"2018-05-29T21:36:17Z","closed_at":"2018-05-29T21:36:17Z","author_association":"NONE","body":"Hi all I have an encoding problem with codeigniter. When I read my database, it shows me special characters of the type Ã©ðŸ”¥ ðŸ”¥.\r\nCan you tell me how to avoid this?","score":1.0},{"url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5509","repository_url":"https://api.github.com/repos/bcit-ci/CodeIgniter","labels_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5509/labels{/name}","comments_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5509/comments","events_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5509/events","html_url":"https://github.com/bcit-ci/CodeIgniter/issues/5509","id":326453865,"node_id":"MDU6SXNzdWUzMjY0NTM4NjU=","number":5509,"title":"Form validation piping not working as expeted","user":{"login":"nikolayanchev","id":3208072,"node_id":"MDQ6VXNlcjMyMDgwNzI=","avatar_url":"https://avatars2.githubusercontent.com/u/3208072?v=4","gravatar_id":"","url":"https://api.github.com/users/nikolayanchev","html_url":"https://github.com/nikolayanchev","followers_url":"https://api.github.com/users/nikolayanchev/followers","following_url":"https://api.github.com/users/nikolayanchev/following{/other_user}","gists_url":"https://api.github.com/users/nikolayanchev/gists{/gist_id}","starred_url":"https://api.github.com/users/nikolayanchev/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/nikolayanchev/subscriptions","organizations_url":"https://api.github.com/users/nikolayanchev/orgs","repos_url":"https://api.github.com/users/nikolayanchev/repos","events_url":"https://api.github.com/users/nikolayanchev/events{/privacy}","received_events_url":"https://api.github.com/users/nikolayanchev/received_events","type":"User","site_admin":false},"labels":[{"id":144140,"node_id":"MDU6TGFiZWwxNDQxNDA=","url":"https://api.github.com/repos/bcit-ci/CodeIgniter/labels/Duplicate","name":"Duplicate","color":"DDDDDD","default":false}],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":1,"created_at":"2018-05-25T09:45:37Z","updated_at":"2018-06-01T14:21:54Z","closed_at":"2018-05-25T10:21:54Z","author_association":"NONE","body":"I have  recently noticed that the piping of form validation rules doesn't work as expected by the term piping, the ordering of called rules is scrambled. for example if you have rules set up like that \r\n`trim|required|xss_clean|exact_length[7]|alpha_numeric|callback_checkCode|callback_checkUsedCode`\r\nthe call sequence comes out like this \r\n`   [0] => trim\r\n    [1] => xss_clean\r\n    [2] => callback_checkCode\r\n    [3] => callback_checkUsedCode\r\n    [4] => required\r\n    [5] => exact_length[7]\r\n    [6] => alpha_numeric\r\n`\r\nShouldn't the sequence be as defined in the string?\r\n","score":1.0},{"url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5506","repository_url":"https://api.github.com/repos/bcit-ci/CodeIgniter","labels_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5506/labels{/name}","comments_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5506/comments","events_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5506/events","html_url":"https://github.com/bcit-ci/CodeIgniter/issues/5506","id":325498641,"node_id":"MDU6SXNzdWUzMjU0OTg2NDE=","number":5506,"title":"set_cookie()","user":{"login":"emad22552","id":8358028,"node_id":"MDQ6VXNlcjgzNTgwMjg=","avatar_url":"https://avatars1.githubusercontent.com/u/8358028?v=4","gravatar_id":"","url":"https://api.github.com/users/emad22552","html_url":"https://github.com/emad22552","followers_url":"https://api.github.com/users/emad22552/followers","following_url":"https://api.github.com/users/emad22552/following{/other_user}","gists_url":"https://api.github.com/users/emad22552/gists{/gist_id}","starred_url":"https://api.github.com/users/emad22552/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/emad22552/subscriptions","organizations_url":"https://api.github.com/users/emad22552/orgs","repos_url":"https://api.github.com/users/emad22552/repos","events_url":"https://api.github.com/users/emad22552/events{/privacy}","received_events_url":"https://api.github.com/users/emad22552/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":1,"created_at":"2018-05-22T23:32:53Z","updated_at":"2018-05-26T20:16:39Z","closed_at":"2018-05-26T20:16:39Z","author_association":"NONE","body":"hi.i think there is a bug in codeigniter3.1.8 in system/core/input.php , set_cookie function, in line 408, where i think `$prefix.$name` should be `$name` in setcookie function.","score":1.0},{"url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5504","repository_url":"https://api.github.com/repos/bcit-ci/CodeIgniter","labels_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5504/labels{/name}","comments_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5504/comments","events_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5504/events","html_url":"https://github.com/bcit-ci/CodeIgniter/issues/5504","id":325020831,"node_id":"MDU6SXNzdWUzMjUwMjA4MzE=","number":5504,"title":"rand() with order_by takes too much time Database","user":{"login":"thesaadmirza","id":14211943,"node_id":"MDQ6VXNlcjE0MjExOTQz","avatar_url":"https://avatars0.githubusercontent.com/u/14211943?v=4","gravatar_id":"","url":"https://api.github.com/users/thesaadmirza","html_url":"https://github.com/thesaadmirza","followers_url":"https://api.github.com/users/thesaadmirza/followers","following_url":"https://api.github.com/users/thesaadmirza/following{/other_user}","gists_url":"https://api.github.com/users/thesaadmirza/gists{/gist_id}","starred_url":"https://api.github.com/users/thesaadmirza/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/thesaadmirza/subscriptions","organizations_url":"https://api.github.com/users/thesaadmirza/orgs","repos_url":"https://api.github.com/users/thesaadmirza/repos","events_url":"https://api.github.com/users/thesaadmirza/events{/privacy}","received_events_url":"https://api.github.com/users/thesaadmirza/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":1,"created_at":"2018-05-21T18:50:50Z","updated_at":"2018-05-21T19:42:32Z","closed_at":"2018-05-21T19:42:32Z","author_association":"NONE","body":"I have a listing of like 1M and when i try $this->db->order-by(\"rand()\");\r\nit takes too much time ,sometimes crashes my server.","score":1.0},{"url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5501","repository_url":"https://api.github.com/repos/bcit-ci/CodeIgniter","labels_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5501/labels{/name}","comments_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5501/comments","events_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5501/events","html_url":"https://github.com/bcit-ci/CodeIgniter/issues/5501","id":323790633,"node_id":"MDU6SXNzdWUzMjM3OTA2MzM=","number":5501,"title":"Log.php has missing directory delimiter","user":{"login":"jlpoolen","id":227313,"node_id":"MDQ6VXNlcjIyNzMxMw==","avatar_url":"https://avatars1.githubusercontent.com/u/227313?v=4","gravatar_id":"","url":"https://api.github.com/users/jlpoolen","html_url":"https://github.com/jlpoolen","followers_url":"https://api.github.com/users/jlpoolen/followers","following_url":"https://api.github.com/users/jlpoolen/following{/other_user}","gists_url":"https://api.github.com/users/jlpoolen/gists{/gist_id}","starred_url":"https://api.github.com/users/jlpoolen/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/jlpoolen/subscriptions","organizations_url":"https://api.github.com/users/jlpoolen/orgs","repos_url":"https://api.github.com/users/jlpoolen/repos","events_url":"https://api.github.com/users/jlpoolen/events{/privacy}","received_events_url":"https://api.github.com/users/jlpoolen/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":3,"created_at":"2018-05-16T21:18:55Z","updated_at":"2018-05-17T15:03:58Z","closed_at":"2018-05-17T15:03:36Z","author_association":"NONE","body":"Log.php has at 187:\r\n\r\n`$filepath = $this->_log_path.'log-'.date('Y-m-d').'.'.$this->_file_ext;`\r\n\r\nIn Linux, an application, Stikked, using this library just hangs with no error output and nothing in the Apache logs.  What allowed me to move forward was modifying the line by inserting a forward slash:\r\n\r\n`$filepath = $this->_log_path.'/log-'.date('Y-m-d').'.'.$this->_file_ext;`\r\n\r\nI do not know much about PHP, but if there is a directory delimiter variable, it should be placed in where I have placed the forward slash so that the delimiting takes place as desired on any operating system.\r\n\r\nThe result of the ommission of a path delimiter in Linux under an Apache system is that a request just hangs with no error output -- really really confounding.","score":1.0},{"url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5498","repository_url":"https://api.github.com/repos/bcit-ci/CodeIgniter","labels_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5498/labels{/name}","comments_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5498/comments","events_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5498/events","html_url":"https://github.com/bcit-ci/CodeIgniter/issues/5498","id":321905894,"node_id":"MDU6SXNzdWUzMjE5MDU4OTQ=","number":5498,"title":"call to a member function quote() on a non-object?","user":{"login":"tonyxyl","id":12979723,"node_id":"MDQ6VXNlcjEyOTc5NzIz","avatar_url":"https://avatars3.githubusercontent.com/u/12979723?v=4","gravatar_id":"","url":"https://api.github.com/users/tonyxyl","html_url":"https://github.com/tonyxyl","followers_url":"https://api.github.com/users/tonyxyl/followers","following_url":"https://api.github.com/users/tonyxyl/following{/other_user}","gists_url":"https://api.github.com/users/tonyxyl/gists{/gist_id}","starred_url":"https://api.github.com/users/tonyxyl/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/tonyxyl/subscriptions","organizations_url":"https://api.github.com/users/tonyxyl/orgs","repos_url":"https://api.github.com/users/tonyxyl/repos","events_url":"https://api.github.com/users/tonyxyl/events{/privacy}","received_events_url":"https://api.github.com/users/tonyxyl/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":1,"created_at":"2018-05-10T11:55:36Z","updated_at":"2018-06-01T14:03:11Z","closed_at":"2018-06-01T14:03:10Z","author_association":"NONE","body":"my app runs on php5.6 and codeigniter 3.0.6 without error for almost 2 years, but recently i found error   like the title says.\r\n\r\nmore detail below:\r\n\r\n```\r\n2018/05/10 19:54:27 [error] 13527#0: *21015224 FastCGI sent in stderr: \"PHP message: PHP Fatal error:  Call to a member function quote() on a non-object in /opt/www/ci/system/database/drivers/pdo/pdo_driver.php on line 231\r\nPHP message: PHP Stack trace:\r\nPHP message: PHP   1. {main}() /opt/www/ci/index.php:0\r\nPHP message: PHP   2. require_once() /opt/www/ci/index.php:317\r\nPHP message: PHP   3. call_user_func_array:{/opt/www/ci/system/core/CodeIgniter.php:514}() /opt/www/ci/system/core/CodeIgniter.php:514\r\nPHP message: PHP   4. Api->newsCalendar() /opt/www/ci/system/core/CodeIgniter.php:514\r\nPHP message: PHP   5. Api_model->getCalendarByDate() /opt/www/ci/application/controllers/Gdfp.php:243\r\nPHP message: PHP   6. CI_DB_query_builder->where() /opt/www/ci/application/models/Api_model.php:154\r\nPHP message: PHP   7. CI_DB_query_builder->_wh() /opt/www/ci/system/database/DB_query_builder.php:609\r\nPHP message: PHP   8. CI_DB_driver->escape() /opt/www/ci/system/database/DB_query_builder.php:669\r\nPHP message: PHP   9. CI_DB_driver->escape_str() /opt/www/ci/system/database/DB_driver.php:1107\r\nPHP message: PHP  10. CI_DB_pdo_driver->_escape_str() /opt/www/ci/system/database/DB_driver.php:1142\"\r\n```","score":1.0},{"url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5497","repository_url":"https://api.github.com/repos/bcit-ci/CodeIgniter","labels_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5497/labels{/name}","comments_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5497/comments","events_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5497/events","html_url":"https://github.com/bcit-ci/CodeIgniter/issues/5497","id":320776283,"node_id":"MDU6SXNzdWUzMjA3NzYyODM=","number":5497,"title":"unable to add multiple where clause in codeigniter batch_update","user":{"login":"singhDevloper","id":39054062,"node_id":"MDQ6VXNlcjM5MDU0MDYy","avatar_url":"https://avatars1.githubusercontent.com/u/39054062?v=4","gravatar_id":"","url":"https://api.github.com/users/singhDevloper","html_url":"https://github.com/singhDevloper","followers_url":"https://api.github.com/users/singhDevloper/followers","following_url":"https://api.github.com/users/singhDevloper/following{/other_user}","gists_url":"https://api.github.com/users/singhDevloper/gists{/gist_id}","starred_url":"https://api.github.com/users/singhDevloper/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/singhDevloper/subscriptions","organizations_url":"https://api.github.com/users/singhDevloper/orgs","repos_url":"https://api.github.com/users/singhDevloper/repos","events_url":"https://api.github.com/users/singhDevloper/events{/privacy}","received_events_url":"https://api.github.com/users/singhDevloper/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":1,"created_at":"2018-05-07T11:55:03Z","updated_at":"2018-05-07T13:28:21Z","closed_at":"2018-05-07T13:28:21Z","author_association":"NONE","body":"Hi there\r\nm trying to update multiple records with batch_update, but it shows me error \r\n'A Database Error Occurred\r\nOne or more rows submitted for batch updating is missing the specified index.'\r\n\r\nhelp me to fix it\r\n\r\nBelow is my code\r\n\r\npublic function batchUpdate ( $table, $data, $condition )\r\n    {\r\n        $this->db->where ( [ 'user_id', 'product_id' ] );\r\n        if ( $this->db->update_batch ( $table, $data, $condition ) )\r\n        {\r\n            return true;\r\n        }\r\n\r\n    }\r\n\r\n\r\n","score":1.0},{"url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5496","repository_url":"https://api.github.com/repos/bcit-ci/CodeIgniter","labels_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5496/labels{/name}","comments_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5496/comments","events_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5496/events","html_url":"https://github.com/bcit-ci/CodeIgniter/issues/5496","id":320586399,"node_id":"MDU6SXNzdWUzMjA1ODYzOTk=","number":5496,"title":"Issue with Codeignter3 with network calls functions","user":{"login":"Jamalc0m","id":8554119,"node_id":"MDQ6VXNlcjg1NTQxMTk=","avatar_url":"https://avatars3.githubusercontent.com/u/8554119?v=4","gravatar_id":"","url":"https://api.github.com/users/Jamalc0m","html_url":"https://github.com/Jamalc0m","followers_url":"https://api.github.com/users/Jamalc0m/followers","following_url":"https://api.github.com/users/Jamalc0m/following{/other_user}","gists_url":"https://api.github.com/users/Jamalc0m/gists{/gist_id}","starred_url":"https://api.github.com/users/Jamalc0m/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/Jamalc0m/subscriptions","organizations_url":"https://api.github.com/users/Jamalc0m/orgs","repos_url":"https://api.github.com/users/Jamalc0m/repos","events_url":"https://api.github.com/users/Jamalc0m/events{/privacy}","received_events_url":"https://api.github.com/users/Jamalc0m/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":3,"created_at":"2018-05-06T11:58:17Z","updated_at":"2018-05-14T07:42:18Z","closed_at":"2018-05-14T07:42:18Z","author_association":"NONE","body":"Hello,\r\n\r\nI haver tried both functions on CI2 and CI3 and found there is an issue with CI3.\r\nOn welcome controller I have added index2 method and execute it.\r\npublic function index()\r\n\t{\r\n\t\t$this->load->view('welcome_message');\r\n\t}\r\n\r\npublic function index2()\r\n {\r\n\t  for($j=0;$j<=10;$j++){\r\n\t  echo file_get_contents(\"https://google.com\");\r\n\t}\r\n}\r\nWhile the index2 is executing on CI3, I wasn't able to access any other function on the controller unless it finishes the execution, When I access index() as example the browser keep waiting until the execution of index2 finishes the it load the page.\r\n\r\nI tried same to do the same thing on CI2 and it worked without any issue.\r\n\r\nCould you please check what could be the issue.\r\nThanks","score":1.0},{"url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5494","repository_url":"https://api.github.com/repos/bcit-ci/CodeIgniter","labels_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5494/labels{/name}","comments_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5494/comments","events_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5494/events","html_url":"https://github.com/bcit-ci/CodeIgniter/issues/5494","id":319783880,"node_id":"MDU6SXNzdWUzMTk3ODM4ODA=","number":5494,"title":"Order by missing when use  count_all_results() along with start_cache()","user":{"login":"anggadarkprince","id":8996500,"node_id":"MDQ6VXNlcjg5OTY1MDA=","avatar_url":"https://avatars3.githubusercontent.com/u/8996500?v=4","gravatar_id":"","url":"https://api.github.com/users/anggadarkprince","html_url":"https://github.com/anggadarkprince","followers_url":"https://api.github.com/users/anggadarkprince/followers","following_url":"https://api.github.com/users/anggadarkprince/following{/other_user}","gists_url":"https://api.github.com/users/anggadarkprince/gists{/gist_id}","starred_url":"https://api.github.com/users/anggadarkprince/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/anggadarkprince/subscriptions","organizations_url":"https://api.github.com/users/anggadarkprince/orgs","repos_url":"https://api.github.com/users/anggadarkprince/repos","events_url":"https://api.github.com/users/anggadarkprince/events{/privacy}","received_events_url":"https://api.github.com/users/anggadarkprince/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":1,"created_at":"2018-05-03T04:18:24Z","updated_at":"2018-06-01T13:56:23Z","closed_at":"2018-06-01T13:56:23Z","author_association":"NONE","body":"I found changes in `DB_query_bulder`, the lines below rise another problem if we use query caching.\r\n\r\n```\r\nif ( ! empty($this->qb_orderby)) {\r\n    $orderby = $this->qb_orderby;\r\n    $this->qb_orderby = NULL;\r\n}\r\n```\r\nbecome\r\n\r\n\r\n```\r\n// ORDER BY usage is often problematic here (most notably\r\n// on Microsoft SQL Server) and ultimately unnecessary\r\n// for selecting COUNT(*) ...\r\n$qb_orderby       = $this->qb_orderby;\r\n$qb_cache_orderby = $this->qb_cache_orderby;\r\n$this->qb_orderby = $this->qb_cache_orderby = NULL;\r\n```\r\nMy sorting method is missing after count total record before filter for pagination\r\n\r\n```\r\n$this->db->start_cache();\r\n$this->from(....)->join(...)->where(...)->order_by(id, desc);\r\n$this->db->stop_cache();\r\n\r\n$dataTotal = $overtimes->count_all_results();\r\n$dataPerPage = $overtimes->limit($length, $start);\r\n$data = $dataPerPage->get()->result_array();\r\n\r\n```\r\n\r\nHow we force keep order_by after using `count_all_result()`?","score":1.0},{"url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5493","repository_url":"https://api.github.com/repos/bcit-ci/CodeIgniter","labels_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5493/labels{/name}","comments_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5493/comments","events_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5493/events","html_url":"https://github.com/bcit-ci/CodeIgniter/issues/5493","id":319676257,"node_id":"MDU6SXNzdWUzMTk2NzYyNTc=","number":5493,"title":"Error when passing an empty array as binds to function db->query","user":{"login":"etcho","id":243427,"node_id":"MDQ6VXNlcjI0MzQyNw==","avatar_url":"https://avatars1.githubusercontent.com/u/243427?v=4","gravatar_id":"","url":"https://api.github.com/users/etcho","html_url":"https://github.com/etcho","followers_url":"https://api.github.com/users/etcho/followers","following_url":"https://api.github.com/users/etcho/following{/other_user}","gists_url":"https://api.github.com/users/etcho/gists{/gist_id}","starred_url":"https://api.github.com/users/etcho/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/etcho/subscriptions","organizations_url":"https://api.github.com/users/etcho/orgs","repos_url
8000
":"https://api.github.com/users/etcho/repos","events_url":"https://api.github.com/users/etcho/events{/privacy}","received_events_url":"https://api.github.com/users/etcho/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":1,"created_at":"2018-05-02T19:08:59Z","updated_at":"2018-06-01T13:53:30Z","closed_at":"2018-06-01T13:53:30Z","author_association":"NONE","body":"I got an error when trying to pass an empty array as binds to the function query. Debugging, I found that the problem occurs if my query have and exclamation mark that it's not a bind mark. In my case it's an url.\r\n`SELECT CONCAT('http://example.com/index.php?op=3')`\r\n\r\nBesides that, I pass an empty array as binds (empty because I build the bind variable dinamically and the array starts as empty). So, my code is something like this:\r\n`$sql = \"SELECT CONCAT('http://example.com/index.php?op=3')\";\r\n$this->db->query($sql, array());`\r\n\r\nRunning this, I got a loop of notices in a while that never ends. It's the while at the end of function compile_binds at DB_driver.php. For some reason, the `$c` variable is zero, and the `$c--` at the begining of the code makes the while never stops.\r\n\r\nIn this cennario I have a query with no valid bind marks and an empty array as bind. This is the error case. If I pass null as binds, it works (it's the workaround that I used to solve my problem), but the bug exists.\r\n\r\nI think that changing the do...while to while...do will work, since the verification would happen first and `$c` wouldn't be lesser than zero.","score":1.0},{"url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5492","repository_url":"https://api.github.com/repos/bcit-ci/CodeIgniter","labels_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5492/labels{/name}","comments_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5492/comments","events_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5492/events","html_url":"https://github.com/bcit-ci/CodeIgniter/issues/5492","id":319605699,"node_id":"MDU6SXNzdWUzMTk2MDU2OTk=","number":5492,"title":"xss_clean","user":{"login":"mfunollet","id":16797068,"node_id":"MDQ6VXNlcjE2Nzk3MDY4","avatar_url":"https://avatars3.githubusercontent.com/u/16797068?v=4","gravatar_id":"","url":"https://api.github.com/users/mfunollet","html_url":"https://github.com/mfunollet","followers_url":"https://api.github.com/users/mfunollet/followers","following_url":"https://api.github.com/users/mfunollet/following{/other_user}","gists_url":"https://api.github.com/users/mfunollet/gists{/gist_id}","starred_url":"https://api.github.com/users/mfunollet/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/mfunollet/subscriptions","organizations_url":"https://api.github.com/users/mfunollet/orgs","repos_url":"https://api.github.com/users/mfunollet/repos","events_url":"https://api.github.com/users/mfunollet/events{/privacy}","received_events_url":"https://api.github.com/users/mfunollet/received_events","type":"User","site_admin":false},"labels":[{"id":144140,"node_id":"MDU6TGFiZWwxNDQxNDA=","url":"https://api.github.com/repos/bcit-ci/CodeIgniter/labels/Duplicate","name":"Duplicate","color":"DDDDDD","default":false}],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":3,"created_at":"2018-05-02T15:38:26Z","updated_at":"2018-06-01T13:57:51Z","closed_at":"2018-06-01T13:57:46Z","author_association":"NONE","body":"We have and issue when apply form validation filter xss_clean.\r\nIf the post is for example: \"\"We are 25% Cat\", after pass the filter, the return value is: \"We are 25�t\".\r\nOnly happend with \"% c\".\r\nWhy?","score":1.0},{"url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5489","repository_url":"https://api.github.com/repos/bcit-ci/CodeIgniter","labels_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5489/labels{/name}","comments_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5489/comments","events_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5489/events","html_url":"https://github.com/bcit-ci/CodeIgniter/issues/5489","id":319355849,"node_id":"MDU6SXNzdWUzMTkzNTU4NDk=","number":5489,"title":"ODBC Drivers Error on Linux ubuntu ","user":{"login":"didinkaj","id":22518184,"node_id":"MDQ6VXNlcjIyNTE4MTg0","avatar_url":"https://avatars1.githubusercontent.com/u/22518184?v=4","gravatar_id":"","url":"https://api.github.com/users/didinkaj","html_url":"https://github.com/didinkaj","followers_url":"https://api.github.com/users/didinkaj/followers","following_url":"https://api.github.com/users/didinkaj/following{/other_user}","gists_url":"https://api.github.com/users/didinkaj/gists{/gist_id}","starred_url":"https://api.github.com/users/didinkaj/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/didinkaj/subscriptions","organizations_url":"https://api.github.com/users/didinkaj/orgs","repos_url":"https://api.github.com/users/didinkaj/repos","events_url":"https://api.github.com/users/didinkaj/events{/privacy}","received_events_url":"https://api.github.com/users/didinkaj/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":1,"created_at":"2018-05-01T22:59:46Z","updated_at":"2018-06-01T13:50:26Z","closed_at":"2018-06-01T13:50:26Z","author_association":"NONE","body":"am able to connect to a .mdb file  source in linux but when i try running a query i.e.\r\n\r\n**model function**\r\n`\tpublic function getDepartmentsAccess() {\r\n\t\t$sql = ('SELECT * FROM DEPARTMENTS ');\r\n\t\t$query = $this -> zkteco -> query($sql);\r\n\r\n\t\treturn $query -> result_array();\r\n\r\n\t}`\r\n\r\n**View results from controller**\r\n```\r\n<?php\r\n\t\t\t\t\tforeach ($leavetypedb as $key => $value) {\r\n\t\t\t\t\t\t\r\n\t\t\t\t\t\techo'<option value=\"'.$value['DEPTNAME'].'\" '.set_select('department', ''.$value['DEPTNAME'].'').'>'.$value['DEPTNAME'].'</option>';\r\n\t\t\t\t\t}\r\n\t\t\t\t\t\t\t\t\t\t\r\n\t\t\t\t\t?>\t\r\n```\r\n# ERROR\r\n\r\nA PHP Error was encountered\r\n\r\nSeverity: Error\r\n\r\nMessage: Out of memory (allocated 2097152) (tried to allocate 18446744069414584352 bytes)\r\n\r\nFilename: odbc/odbc_result.php\r\n\r\nLine Number: 171\r\n\r\nBacktrace:\r\n\r\n\r\n\r\n# While the code below works fine without an error\r\n`<?php\r\n\t\t\t\t\t\t$query = 'SELECT * FROM DEPARTMENTS';\r\n\t\t\t\t\t\t$mdb_file = '/var/www/html/imani.com/public_html/lms/ZKAccess.mdb';\r\n\t\t\t\t\t\t$uname = explode(\" \",php_uname());\r\n\t\t\t\t\t\t$os = $uname[0];\r\n\r\n\t\t\t\t\t\techo$os;\r\n\t\t\t\t\t\tswitch ($os){\r\n\t\t\t\t\t\t  case 'Windows':\r\n\t\t\t\t\t\t\t$driver = '{Microsoft Access Driver (*.mdb)}';\r\n\t\t\t\t\t\t\tbreak;\r\n\t\t\t\t\t\t  case 'Linux':\r\n\t\t\t\t\t\t\t$driver = 'MDBTools';\r\n\t\t\t\t\t\t\tbreak;\r\n\t\t\t\t\t\t  default:\r\n\t\t\t\t\t\t\texit(\"Don't know about this OS\");\r\n\t\t\t\t\t\t}\r\n\t\t\t\t\t\t$dataSourceName = \"odbc:Driver=$driver;DBQ=$mdb_file;\";\r\n\r\n\t\t\t\t\t\ttry{\r\n\t\t\t\t\t\t$connection = new PDO($dataSourceName);\r\n\t\t\t\t\t\t}catch(PDOException $exception){\r\n\t\t\t\t\t\t\tdie(\"Unable to open database.Error message:$exception.\");\r\n\t\t\t\t\t\t}\r\n\t\t\t\t\t\t$result = $connection->query($query)->fetchAll(\\PDO::FETCH_ASSOC);\r\n\t\t\t\t\t\t\r\n\r\n\r\n\r\n\t\t\t\t\tforeach ($result as $key => $value) {\r\n\t\t\t\t\t\t\r\n\t\t\t\t\t\techo'<option value=\"'.$value['DEPTNAME'].'\" '.set_select('department', ''.$value['DEPTNAME'].'').'>'.$value['DEPTNAME'].'</option>';\r\n\t\t\t\t\t}\r\n\t\t\t\t\t\t\t\t\t\t\r\n\t\t\t\t\t?>\t\t\t\t`","score":1.0},{"url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5487","repository_url":"https://api.github.com/repos/bcit-ci/CodeIgniter","labels_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5487/labels{/name}","comments_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5487/comments","events_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5487/events","html_url":"https://github.com/bcit-ci/CodeIgniter/issues/5487","id":319044546,"node_id":"MDU6SXNzdWUzMTkwNDQ1NDY=","number":5487,"title":"Session Library: Typing mistake in comment above unmark_temp function","user":{"login":"trulyfurqan","id":22556762,"node_id":"MDQ6VXNlcjIyNTU2NzYy","avatar_url":"https://avatars1.githubusercontent.com/u/22556762?v=4","gravatar_id":"","url":"https://api.github.com/users/trulyfurqan","html_url":"https://github.com/trulyfurqan","followers_url":"https://api.github.com/users/trulyfurqan/followers","following_url":"https://api.github.com/users/trulyfurqan/following{/other_user}","gists_url":"https://api.github.com/users/trulyfurqan/gists{/gist_id}","starred_url":"https://api.github.com/users/trulyfurqan/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/trulyfurqan/subscriptions","organizations_url":"https://api.github.com/users/trulyfurqan/orgs","repos_url":"https://api.github.com/users/trulyfurqan/repos","events_url":"https://api.github.com/users/trulyfurqan/events{/privacy}","received_events_url":"https://api.github.com/users/trulyfurqan/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":0,"created_at":"2018-04-30T21:36:56Z","updated_at":"2018-05-01T01:23:04Z","closed_at":"2018-05-01T01:23:04Z","author_association":"NONE","body":"In Session library there is a typing mistake in comment above \"unmark_temp\" function.\r\n\r\n\r\n**Current Comment:**\r\n```\r\n/**\r\n * Unmark flash\r\n *\r\n * @param\tmixed\t$key\tSession data key(s)\r\n * @return\tvoid\r\n */\r\n```\r\n\r\n\r\n**Suggested Comment:**\r\n```\r\n/**\r\n * Unmark temp\r\n *\r\n * @param\tmixed\t$key\tSession data key(s)\r\n * @return\tvoid\r\n */\r\n```","score":1.0},{"url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5486","repository_url":"https://api.github.com/repos/bcit-ci/CodeIgniter","labels_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5486/labels{/name}","comments_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5486/comments","events_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5486/events","html_url":"https://github.com/bcit-ci/CodeIgniter/issues/5486","id":319034459,"node_id":"MDU6SXNzdWUzMTkwMzQ0NTk=","number":5486,"title":"Session Library: unset_tempdata function doesn't remove the temporary SESSION variable","user":{"login":"trulyfurqan","id":22556762,"node_id":"MDQ6VXNlcjIyNTU2NzYy","avatar_url":"https://avatars1.githubusercontent.com/u/22556762?v=4","gravatar_id":"","url":"https://api.github.com/users/trulyfurqan","html_url":"https://github.com/trulyfurqan","followers_url":"https://api.github.com/users/trulyfurqan/followers","following_url":"https://api.github.com/users/trulyfurqan/following{/other_user}","gists_url":"https://api.github.com/users/trulyfurqan/gists{/gist_id}","starred_url":"https://api.github.com/users/trulyfurqan/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/trulyfurqan/subscriptions","organizations_url":"https://api.github.com/users/trulyfurqan/orgs","repos_url":"https://api.github.com/users/trulyfurqan/repos","events_url":"https://api.github.com/users/trulyfurqan/events{/privacy}","received_events_url":"https://api.github.com/users/trulyfurqan/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":1,"created_at":"2018-04-30T21:01:28Z","updated_at":"2018-05-01T01:20:06Z","closed_at":"2018-05-01T01:20:06Z","author_association":"NONE","body":"Hi, I'm Furqan.\r\nRecently i face a situation where i have to manually remove the temporary SESSION variable. But the function \"unset_tempdata\" only allows us to remove the marker that makes the specific SESSION variable as tempdata and not actually removes the SESSION variable.\r\n\r\nA question also arises that if both \"unset_tempdata\" and \"unmark_temp\" functions perform the same set of instructions then why we are having two separate functions instead of one.\r\n\r\n\r\n**Current \"unset_tempdata\" function code:**\r\n```\r\npublic function unset_tempdata($key)\r\n{\r\n\t$this->unmark_temp($key);\r\n}\r\n```\r\n\r\n\r\n**My suggested code for \"unset_tempdata\" function:**\r\n```\r\npublic function unset_tempdata($key)\r\n{\r\n\t$this->unmark_temp($key);\r\n\t$this->unset_userdata($key);\r\n}\r\n```\r\n\r\n\r\nIn my suggested code the \"unset_tempdata\" function will remove the marker that makes the specific SESSION variable as tempdata as well as removes the temporary SESSION variable.","score":1.0},{"url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5484","repository_url":"https://api.github.com/repos/bcit-ci/CodeIgniter","labels_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5484/labels{/name}","comments_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5484/comments","events_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5484/events","html_url":"https://github.com/bcit-ci/CodeIgniter/issues/5484","id":318776144,"node_id":"MDU6SXNzdWUzMTg3NzYxNDQ=","number":5484,"title":"php7","user":{"login":"xiaolifeidao2016","id":18713750,"node_id":"MDQ6VXNlcjE4NzEzNzUw","avatar_url":"https://avatars2.githubusercontent.com/u/18713750?v=4","gravatar_id":"","url":"https://api.github.com/users/xiaolifeidao2016","html_url":"https://github.com/xiaolifeidao2016","followers_url":"https://api.github.com/users/xiaolifeidao2016/followers","following_url":"https://api.github.com/users/xiaolifeidao2016/following{/other_user}","gists_url":"https://api.github.com/users/xiaolifeidao2016/gists{/gist_id}","starred_url":"https://api.github.com/users/xiaolifeidao2016/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/xiaolifeidao2016/subscriptions","organizations_url":"https://api.github.com/users/xiaolifeidao2016/orgs","repos_url":"https://api.github.com/users/xiaolifeidao2016/repos","events_url":"https://api.github.com/users/xiaolifeidao2016/events{/privacy}","received_events_url":"https://api.github.com/users/xiaolifeidao2016/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":1,"created_at":"2018-04-30T03:07:02Z","updated_at":"2018-04-30T04:26:47Z","closed_at":"2018-04-30T04:26:47Z","author_association":"NONE","body":"CI什么时候基于PHP7版本的开发呢？","score":1.0},{"url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5483","repository_url":"https://api.github.com/repos/bcit-ci/CodeIgniter","labels_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5483/labels{/name}","comments_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5483/comments","events_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5483/events","html_url":"https://github.com/bcit-ci/CodeIgniter/issues/5483","id":318116520,"node_id":"MDU6SXNzdWUzMTgxMTY1MjA=","number":5483,"title":"Form Validation callbacks do not work within models.","user":{"login":"EoghanH","id":436600,"node_id":"MDQ6VXNlcjQzNjYwMA==","avatar_url":"https://avatars1.githubusercontent.com/u/436600?v=4","gravatar_id":"","url":"https://api.github.com/users/EoghanH","html_url":"https://github.com/EoghanH","followers_url":"https://api.github.com/users/EoghanH/followers","following_url":"https://api.github.com/users/EoghanH/following{/other_user}","gists_url":"https://api.github.com/users/EoghanH/gists{/gist_id}","starred_url":"https://api.github.com/users/EoghanH/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/EoghanH/subscriptions","organizations_url":"https://api.github.com/users/EoghanH/orgs","repos_url":"https://api.github.com/users/EoghanH/repos","events_url":"https://api.github.com/users/EoghanH/events{/privacy}","received_events_url":"https://api.github.com/users/EoghanH/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":2,"created_at":"2018-04-26T16:57:56Z","updated_at":"2018-04-27T09:13:43Z","closed_at":"2018-04-27T09:13:43Z","author_association":"NONE","body":"```\r\nautoload.php : \r\n$autoload['libraries'] = array('form_validation', 'session');\r\n$autoload['helper'] = array('url', 'form', 'security');\r\n```\r\n\r\nCode in controller;\r\n\r\n```\r\n$this->form_validation->set_rules('custom_id', 'custom_id', array('required', 'numeric', array($this->Services_model, 'valid_custom_id')));\r\n\r\nif ($this->form_validation->run() === false) {\r\n            var_dump($this->form_validation->error_array());\r\n        }\r\n```\r\n\r\nCode in Services_model:\r\n```\r\n\r\npublic function valid_custom_id($custom_id) {\r\n        $this->form_validation->set_message('valid_custom_id', 'custom_id test.');\r\n        return false;\r\n    }\r\n```\r\n\r\nThis will return back with the error;\r\n\r\n`'custom_id' => string 'Unable to access an error message corresponding to your field name custom_id.(Anonymous function)'`\r\n\r\nIf the controller is swapped over to using `callback_valid_custom_id`, and the function is moved from the model to the controller, it works as expected. Similarly, if the callback function always returns with `true`, then no error is reported - but that defeats the purpose.\r\n\r\nAccording to the documentation available at https://www.codeigniter.com/userguide3/libraries/form_validation.html#callable-use-anything-as-a-rule , the above should work as expected in all cases.","score":1.0},{"url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5482","repository_url":"https://api.github.com/repos/bcit-ci/CodeIgniter","labels_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5482/labels{/name}","comments_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5482/comments","events_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5482/events","html_url":"https://github.com/bcit-ci/CodeIgniter/issues/5482","id":317577360,"node_id":"MDU6SXNzdWUzMTc1NzczNjA=","number":5482,"title":"is_countable conflict with PHP 7.3","user":{"login":"DavidTurnerDotInfo","id":3332249,"node_id":"MDQ6VXNlcjMzMzIyNDk=","avatar_url":"https://avatars3.githubusercontent.com/u/3332249?v=4","gravatar_id":"","url":"https://api.github.com/users/DavidTurnerDotInfo","html_url":"https://github.com/DavidTurnerDotInfo","followers_url":"https://api.github.com/users/DavidTurnerDotInfo/followers","following_url":"https://api.github.com/users/DavidTurnerDotInfo/following{/other_user}","gists_url":"https://api.github.com/users/DavidTurnerDotInfo/gists{/gist_id}","starred_url":"https://api.github.com/users/DavidTurnerDotInfo/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/DavidTurnerDotInfo/subscriptions","organizations_url":"https://api.github.com/users/DavidTurnerDotInfo/orgs","repos_url":"https://api.github.com/users/DavidTurnerDotInfo/repos","events_url":"https://api.github.com/users/DavidTurnerDotInfo/events{/privacy}","received_events_url":"https://api.github.com/users/DavidTurnerDotInfo/received_events","type":"User","site_admin":false},"labels":[{"id":144139,"node_id":"MDU6TGFiZWwxNDQxMzk=","url":"https://api.github.com/repos/bcit-ci/CodeIgniter/labels/Looking%20Into%20It","name":"Looking Into It","color":"02e10c","default":false}],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":{"url":"https://api.github.com/repos/bcit-ci/CodeIgniter/milestones/4","html_url":"https://github.com/bcit-ci/CodeIgniter/milestone/4","labels_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/milestones/4/labels","id":1212589,"node_id":"MDk6TWlsZXN0b25lMTIxMjU4OQ==","number":4,"title":"3.2.0","description":"","creator":{"login":"narfbg","id":1058011,"node_id":"MDQ6VXNlcjEwNTgwMTE=","avatar_url":"https://avatars0.githubusercontent.com/u/1058011?v=4","gravatar_id":"","url":"https://api.github.com/users/narfbg","html_url":"https://github.com/narfbg","followers_url":"https://api.github.com/users/narfbg/followers","following_url":"https://api.github.com/users/narfbg/following{/other_user}","gists_url":"https://api.github.com/users/narfbg/gists{/gist_id}","starred_url":"https://api.github.com/users/narfbg/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/narfbg/subscriptions","organizations_url":"https://api.github.com/users/narfbg/orgs","repos_url":"https://api.github.com/users/narfbg/repos","events_url":"https://api.github.com/users/narfbg/events{/privacy}","received_events_url":"https://api.github.com/users/narfbg/received_events","type":"User","site_admin":false},"open_issues":7,"closed_issues":60,"state":"open","created_at":"2015-07-17T09:35:31Z","updated_at":"2018-06-01T15:08:10Z","due_on":null,"closed_at":null},"comments":6,"created_at":"2018-04-25T10:56:18Z","updated_at":"2018-06-01T15:08:10Z","closed_at":"2018-06-01T15:08:10Z","author_association":"NONE","body":"https://wiki.php.net/rfc/is-countable\r\n\r\nPHP 7.3 includes a function called is_countable that conflicts with the function in the CI inflection_helper.php it serves a different purpose as the former checks for whether the input can be assessed by count() and the latter whether the string input is grammatically countable for countable.\r\n\r\nHappy to help resolve this but I'm pretty new to contributing.","score":1.0},{"url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5480","repository_url":"https://api.github.com/repos/bcit-ci/CodeIgniter","labels_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5480/labels{/name}","comments_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5480/comments","events_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5480/events","html_url":"https://github.com/bcit-ci/CodeIgniter/issues/5480","id":317469782,"node_id":"MDU6SXNzdWUzMTc0Njk3ODI=","number":5480,"title":"Input.php param $masklen undefined","user":{"login":"skiy","id":3362147,"node_id":"MDQ6VXNlcjMzNjIxNDc=","avatar_url":"https://avatars0.githubusercontent.com/u/3362147?v=4","gravatar_id":"","url":"https://api.github.com/users/skiy","html_url":"https://github.com/skiy","followers_url":"https://api.github.com/users/skiy/followers","following_url":"https://api.github.com/users/skiy/following{/other_user}","gists_url":"https://api.github.com/users/skiy/gists{/gist_id}","starred_url":"https://api.github.com/users/skiy/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/skiy/subscriptions","organizations_url":"https://api.github.com/users/skiy/orgs","repos_url":"https://api.github.com/users/skiy/repos","events_url":"https://api.github.com/users/skiy/events{/privacy}","received_events_url":"https://api.github.com/users/skiy/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":2,"created_at":"2018-04-25T03:48:55Z","updated_at":"2018-04-30T16:28:42Z","closed_at":"2018-04-30T16:28:42Z","author_association":"NONE","body":"\r\nIn line 514.\r\n```php\r\nsscanf($proxy_ips[$i], '%[^/]/%d', $netaddr, $masklen);\r\n```\r\n\r\n![selection_007](https://user-images.githubusercontent.com/3362147/39225363-858cc306-487e-11e8-95f4-2c74d98c0b72.png)\r\n","score":1.0},{"url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5479","repository_url":"https://api.github.com/repos/bcit-ci/CodeIgniter","labels_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5479/labels{/name}","comments_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5479/comments","events_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5479/events","html_url":"https://github.com/bcit-ci/CodeIgniter/issues/5479","id":317064970,"node_id":"MDU6SXNzdWUzMTcwNjQ5NzA=","number":5479,"title":"Incorrect uri in caching functions","user":{"login":"hamzawain7","id":20171624,"node_id":"MDQ6VXNlcjIwMTcxNjI0","avatar_url":"https://avatars3.githubusercontent.com/u/20171624?v=4","gravatar_id":"","url":"https://api.github.com/users/hamzawain7","html_url":"https://github.com/hamzawain7","followers_url":"https://api.github.com/users/hamzawain7/followers","following_url":"https://api.github.com/users/hamzawain7/following{/other_user}","gists_url":"https://api.github.com/users/hamzawain7/gists{/gist_id}","starred_url":"https://api.github.com/users/hamzawain7/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/hamzawain7/subscriptions","organizations_url":"https://api.github.com/users/hamzawain7/orgs","repos_url":"https://api.github.com/users/hamzawain7/repos","events_url":"https://api.github.com/users/hamzawain7/events{/privacy}","received_events_url":"https://api.github.com/users/hamzawain7/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":{"url":"https://api.github.com/repos/bcit-ci/CodeIgniter/milestones/4","html_url":"https://github.com/bcit-ci/CodeIgniter/milestone/4","labels_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/milestones/4/labels","id":1212589,"node_id":"MDk6TWlsZXN0b25lMTIxMjU4OQ==","number":4,"title":"3.2.0","description":"","creator":{"login":"narfbg","id":1058011,"node_id":"MDQ6VXNlcjEwNTgwMTE=","avatar_url":"https://avatars0.githubusercontent.com/u/1058011?v=4","gravatar_id":"","url":"https://api.github.com/users/narfbg","html_url":"https://github.com/narfbg","followers_url":"https://api.github.com/users/narfbg/followers","following_url":"https://api.github.com/users/narfbg/following{/other_user}","gists_url":"https://api.github.com/users/narfbg/gists{/gist_id}","starred_url":"https://api.github.com/users/narfbg/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/narfbg/subscriptions","organizations_url":"https://api.github.com/users/narfbg/orgs","repos_url":"https://api.github.com/users/narfbg/repos","events_url":"https://api.github.com/users/narfbg/events{/privacy}","received_events_url":"https://api.github.com/users/narfbg/received_events","type":"User","site_admin":false},"open_issues":7,"closed_issues":60,"state":"open","created_at":"2015-07-17T09:35:31Z","updated_at":"2018-06-01T15:08:10Z","due_on":null,"closed_at":null},"comments":8,"created_at":"2018-04-24T04:18:40Z","updated_at":"2018-04-24T13:42:21Z","closed_at":"2018-04-24T13:10:06Z","author_association":"NONE","body":"Output.php\r\n_write_cache line 548. $uri is incorrect\r\n_display_cache line 631. $uri is incorrect\r\n\r\nThe uri formed doesn't has a slash after index_page i.e uri has the value 'http://localhost/index.phpcontroller/function' instead of 'http://localhost/index.php/controller/function'.","score":1.0},{"url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5478","repository_url":"https://api.github.com/repos/bcit-ci/CodeIgniter","labels_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5478/labels{/name}","comments_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5478/comments","events_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5478/events","html_url":"https://github.com/bcit-ci/CodeIgniter/issues/5478","id":316444946,"node_id":"MDU6SXNzdWUzMTY0NDQ5NDY=","number":5478,"title":"system/database/DB.php Syntax check error in eclipse.","user":{"login":"wangchuangen","id":8543567,"node_id":"MDQ6VXNlcjg1NDM1Njc=","avatar_url":"https://avatars3.githubusercontent.com/u/8543567?v=4","gravatar_id":"","url":"https://api.github.com/users/wangchuangen","html_url":"https://github.com/wangchuangen","followers_url":"https://api.github.com/users/wangchuangen/followers","following_url":"https://api.github.com/users/wangchuangen/following{/other_user}","gists_url":"https://api.github.com/users/wangchuangen/gists{/gist_id}","starred_url":"https://api.github.com/users/wangchuangen/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/wangchuangen/subscriptions","organizations_url":"https://api.github.com/users/wangchuangen/orgs","repos_url":"https://api.github.com/users/wangchuangen/repos","events_url":"https://api.github.com/users/wangchuangen/events{/privacy}","received_events_url":"https://api.github.com/users/wangchuangen/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":3,"created_at":"2018-04-21T00:20:37Z","updated_at":"2018-04-23T11:20:44Z","closed_at":"2018-04-23T11:20:44Z","author_association":"NONE","body":"在eclipse中,DB.php提示错误信息\"Cannot declare type CI_DB because the name is already in use\",虽然我知道这是eclipse误报,但看到错误信息心里就是不爽,\r\n我的解决办法是将第二个\"class CI_DB\"写入新的文件,然后在原来的语句处引用新加入的文件,\r\n\r\nIn eclipse, db.php prompts the error message \"Cannot declare type CI_DB because the name is already in use\", although I know that this is eclipse misinformation, but it is not good to see the error message.\r\nMy solution is to write the second \"class CI_DB\" to the new file, and then to reference the newly added file in the original statement,","score":1.0},{"url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5477","repository_url":"https://api.github.com/repos/bcit-ci/CodeIgniter","labels_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5477/labels{/name}","comments_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5477/comments","events_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5477/events","html_url":"https://github.com/bcit-ci/CodeIgniter/issues/5477","id":315816643,"node_id":"MDU6SXNzdWUzMTU4MTY2NDM=","number":5477,"title":"$this->load->model() is very very slow","user":{"login":"bimohxh","id":13518855,"node_id":"MDQ6VXNlcjEzNTE4ODU1","avatar_url":"https://avatars0.githubusercontent.com/u/13518855?v=4","gravatar_id":"","url":"https://api.github.com/users/bimohxh","html_url":"https://github.com/bimohxh","followers_url":"https://api.github.com/users/bimohxh/followers","following_url":"https://api.github.com/users/bimohxh/following{/other_user}","gists_url":"https://api.github.com/users/bimohxh/gists{/gist_id}","starred_url":"https://api.github.com/users/bimohxh/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/bimohxh/subscriptions","organizations_url":"https://api.github.com/users/bimohxh/orgs","repos_url":"https://api.github.com/users/bimohxh/repos","events_url":"https://api.github.com/users/bimohxh/events{/privacy}","received_events_url":"https://api.github.com/users/bimohxh/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":1,"created_at":"2018-04-19T10:22:57Z","updated_at":"2018-04-23T11:30:28Z","closed_at":"2018-04-23T11:30:28Z","author_association":"NONE","body":"my env is \r\n\r\nCodeIgniter verison is :  `3.1.8`\r\nPHP version is : `7.2.4`\r\n\r\nI have a  controller, I fund the  code below is where cost time:\r\n```php\r\necho \"time1\".time().\"\\n\";\r\n$this->load->model($model_path, 'route');\r\necho \"time2\".time().\"\\n\";\r\n```\r\nthe code is waste alomt **30**s, but not error, just slow.\r\n\r\nis the php version not compatible with CI？how can I solve this problem\r\n\r\n\r\n\r\n\r\n\r\n\r\n","score":1.0},{"url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5476","repository_url":"https://api.github.com/repos/bcit-ci/CodeIgniter","labels_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5476/labels{/name}","comments_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5476/comments","events_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5476/events","html_url":"https://github.com/bcit-ci/CodeIgniter/issues/5476","id":315745778,"node_id":"MDU6SXNzdWUzMTU3NDU3Nzg=","number":5476,"title":"Encrypt library ","user":{"login":"mahendravikhar","id":36271747,"node_id":"MDQ6VXNlcjM2MjcxNzQ3","avatar_url":"https://avatars1.githubusercontent.com/u/36271747?v=4","gravatar_id":"","url":"https://api.github.com/users/mahendravikhar","html_url":"https://github.com/mahendravikhar","followers_url":"https://api.github.com/users/mahendravikhar/followers","following_url":"https://api.github.com/users/mahendravikhar/following{/other_user}","gists_url":"https://api.github.com/users/mahendravikhar/gists{/gist_id}","starred_url":"https://api.github.com/users/mahe
2da0
ndravikhar/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/mahendravikhar/subscriptions","organizations_url":"https://api.github.com/users/mahendravikhar/orgs","repos_url":"https://api.github.com/users/mahendravikhar/repos","events_url":"https://api.github.com/users/mahendravikhar/events{/privacy}","received_events_url":"https://api.github.com/users/mahendravikhar/received_events","type":"User","site_admin":false},"labels":[{"id":144140,"node_id":"MDU6TGFiZWwxNDQxNDA=","url":"https://api.github.com/repos/bcit-ci/CodeIgniter/labels/Duplicate","name":"Duplicate","color":"DDDDDD","default":false}],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":1,"created_at":"2018-04-19T06:40:15Z","updated_at":"2018-04-19T08:48:11Z","closed_at":"2018-04-19T08:48:07Z","author_association":"NONE","body":"The Encrypt Library is still using mcrypt extension. How we can use this on PHP 7.1.11\r\n\r\nURL: https://github.com/bcit-ci/CodeIgniter/blob/develop/system/libraries/Encrypt.php\r\n\r\n`public function __construct()\r\n\t{\r\n\t\tif (($this->_mcrypt_exists = function_exists('mcrypt_encrypt')) === FALSE)\r\n\t\t{\r\n\t\t\tshow_error('The Encrypt library requires the Mcrypt extension.');\r\n\t\t}\r\n\t\tlog_message('info', 'Encrypt Class Initialized');\r\n\t}`","score":1.0},{"url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5475","repository_url":"https://api.github.com/repos/bcit-ci/CodeIgniter","labels_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5475/labels{/name}","comments_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5475/comments","events_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5475/events","html_url":"https://github.com/bcit-ci/CodeIgniter/issues/5475","id":315738890,"node_id":"MDU6SXNzdWUzMTU3Mzg4OTA=","number":5475,"title":"Database Restore Issue ","user":{"login":"sanjayacloud","id":34233059,"node_id":"MDQ6VXNlcjM0MjMzMDU5","avatar_url":"https://avatars3.githubusercontent.com/u/34233059?v=4","gravatar_id":"","url":"https://api.github.com/users/sanjayacloud","html_url":"https://github.com/sanjayacloud","followers_url":"https://api.github.com/users/sanjayacloud/followers","following_url":"https://api.github.com/users/sanjayacloud/following{/other_user}","gists_url":"https://api.github.com/users/sanjayacloud/gists{/gist_id}","starred_url":"https://api.github.com/users/sanjayacloud/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/sanjayacloud/subscriptions","organizations_url":"https://api.github.com/users/sanjayacloud/orgs","repos_url":"https://api.github.com/users/sanjayacloud/repos","events_url":"https://api.github.com/users/sanjayacloud/events{/privacy}","received_events_url":"https://api.github.com/users/sanjayacloud/received_events","type":"User","site_admin":false},"labels":[{"id":144140,"node_id":"MDU6TGFiZWwxNDQxNDA=","url":"https://api.github.com/repos/bcit-ci/CodeIgniter/labels/Duplicate","name":"Duplicate","color":"DDDDDD","default":false}],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":1,"created_at":"2018-04-19T06:06:43Z","updated_at":"2018-04-19T08:45:41Z","closed_at":"2018-04-19T08:45:36Z","author_association":"NONE","body":"Hello,\r\nI have create a database backup using codeigniter $this->load->dbutil(); This Work fine and I can download my database backup. But I try to restore this backup and I got below error..\r\n\r\nSQL query:\r\n\r\nINSERT INTO app_user (id, user_name, email, password, role, status) VALUES (10, Admin.sgh, admin@supreme.lk, 202cb962ac59075b964b07152d234b70, 0, 0)\r\nMySQL said: Documentation\r\n\r\n#1064 - You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '@supreme.lk, 202cb962ac59075b964b07152d234b70, 0, 0)' at line 1","score":1.0},{"url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5474","repository_url":"https://api.github.com/repos/bcit-ci/CodeIgniter","labels_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5474/labels{/name}","comments_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5474/comments","events_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5474/events","html_url":"https://github.com/bcit-ci/CodeIgniter/issues/5474","id":315464253,"node_id":"MDU6SXNzdWUzMTU0NjQyNTM=","number":5474,"title":"dbprefix not added to Like Clause in 3.1.8","user":{"login":"DavidTurnerDotInfo","id":3332249,"node_id":"MDQ6VXNlcjMzMzIyNDk=","avatar_url":"https://avatars3.githubusercontent.com/u/3332249?v=4","gravatar_id":"","url":"https://api.github.com/users/DavidTurnerDotInfo","html_url":"https://github.com/DavidTurnerDotInfo","followers_url":"https://api.github.com/users/DavidTurnerDotInfo/followers","following_url":"https://api.github.com/users/DavidTurnerDotInfo/following{/other_user}","gists_url":"https://api.github.com/users/DavidTurnerDotInfo/gists{/gist_id}","starred_url":"https://api.github.com/users/DavidTurnerDotInfo/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/DavidTurnerDotInfo/subscriptions","organizations_url":"https://api.github.com/users/DavidTurnerDotInfo/orgs","repos_url":"https://api.github.com/users/DavidTurnerDotInfo/repos","events_url":"https://api.github.com/users/DavidTurnerDotInfo/events{/privacy}","received_events_url":"https://api.github.com/users/DavidTurnerDotInfo/received_events","type":"User","site_admin":false},"labels":[{"id":144140,"node_id":"MDU6TGFiZWwxNDQxNDA=","url":"https://api.github.com/repos/bcit-ci/CodeIgniter/labels/Duplicate","name":"Duplicate","color":"DDDDDD","default":false}],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":2,"created_at":"2018-04-18T12:46:12Z","updated_at":"2018-04-18T13:10:17Z","closed_at":"2018-04-18T12:57:09Z","author_association":"NONE","body":"My code was working with 3.1.7 but since updating to 3.1.8 the following instruction no longer receives the table prefix in the MySQL output.  \r\n\r\n$this->db->like('Blocks.BlockType',$val);\r\n\r\nI'm using the following config;\r\n$db['dbdriver'] = \"mysqli\"; \r\n$db['dbprefix'] = \"myprefix_\";\r\n$db['pconnect'] = FALSE;\r\n$db['db_debug'] = TRUE;\r\n$db['cache_on'] = FALSE;\r\n$db['cachedir'] = \"\";\r\n$db['char_set'] = \"utf8\";\r\n$db['dbcollat'] = \"utf8_general_ci\";\r\n","score":1.0},{"url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5473","repository_url":"https://api.github.com/repos/bcit-ci/CodeIgniter","labels_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5473/labels{/name}","comments_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5473/comments","events_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5473/events","html_url":"https://github.com/bcit-ci/CodeIgniter/issues/5473","id":315343240,"node_id":"MDU6SXNzdWUzMTUzNDMyNDA=","number":5473,"title":"image_lib class doesn't accept float values for width and height in ci3","user":{"login":"hamzawain7","id":20171624,"node_id":"MDQ6VXNlcjIwMTcxNjI0","avatar_url":"https://avatars3.githubusercontent.com/u/20171624?v=4","gravatar_id":"","url":"https://api.github.com/users/hamzawain7","html_url":"https://github.com/hamzawain7","followers_url":"https://api.github.com/users/hamzawain7/followers","following_url":"https://api.github.com/users/hamzawain7/following{/other_user}","gists_url":"https://api.github.com/users/hamzawain7/gists{/gist_id}","starred_url":"https://api.github.com/users/hamzawain7/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/hamzawain7/subscriptions","organizations_url":"https://api.github.com/users/hamzawain7/orgs","repos_url":"https://api.github.com/users/hamzawain7/repos","events_url":"https://api.github.com/users/hamzawain7/events{/privacy}","received_events_url":"https://api.github.com/users/hamzawain7/received_events","type":"User","site_admin":false},"labels":[{"id":144140,"node_id":"MDU6TGFiZWwxNDQxNDA=","url":"https://api.github.com/repos/bcit-ci/CodeIgniter/labels/Duplicate","name":"Duplicate","color":"DDDDDD","default":false}],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":2,"created_at":"2018-04-18T06:09:59Z","updated_at":"2018-04-18T09:09:53Z","closed_at":"2018-04-18T09:01:49Z","author_association":"NONE","body":"so i upgraded from ci2 to ci3 and my cropping code using image_lib broke because i was sending a calculated float value of width and height key to the initialize function which worked fine but now it doesn't. I know giving a float pixel value doesn't makes sense but it worked in 2.1.3 and now it doesn't work in 3.1.7. I think it has to do with the !ctype_digit((string)$val) check in system/libraries/Image_lib.php. If you don't wana accept decimal values thats fine but at least mention it in the code igniter upgrade docs as it is a breaking change.","score":1.0},{"url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5472","repository_url":"https://api.github.com/repos/bcit-ci/CodeIgniter","labels_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5472/labels{/name}","comments_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5472/comments","events_url":"https://api.github.com/repos/bcit-ci/CodeIgniter/issues/5472/events","html_url":"https://github.com/bcit-ci/CodeIgniter/issues/5472","id":315020494,"node_id":"MDU6SXNzdWUzMTUwMjA0OTQ=","number":5472,"title":"Codeigniter Backup Restore Error","user":{"login":"sanjayacloud","id":34233059,"node_id":"MDQ6VXNlcjM0MjMzMDU5","avatar_url":"https://avatars3.githubusercontent.com/u/34233059?v=4","gravatar_id":"","url":"https://api.github.com/users/sanjayacloud","html_url":"https://github.com/sanjayacloud","followers_url":"https://api.github.com/users/sanjayacloud/followers","following_url":"https://api.github.com/users/sanjayacloud/following{/other_user}","gists_url":"https://api.github.com/users/sanjayacloud/gists{/gist_id}","starred_url":"https://api.github.com/users/sanjayacloud/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/sanjayacloud/subscriptions","organizations_url":"https://api.github.com/users/sanjayacloud/orgs","repos_url":"https://api.github.com/users/sanjayacloud/repos","events_url":"https://api.github.com/users/sanjayacloud/events{/privacy}","received_events_url":"https://api.github.com/users/sanjayacloud/received_events","type":"User","site_admin":false},"labels":[{"id":144140,"node_id":"MDU6TGFiZWwxNDQxNDA=","url":"https://api.github.com/repos/bcit-ci/CodeIgniter/labels/Duplicate","name":"Duplicate","color":"DDDDDD","default":false}],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":1,"created_at":"2018-04-17T11:36:28Z","updated_at":"2018-04-18T09:03:27Z","closed_at":"2018-04-18T09:03:23Z","author_association":"NONE","body":"Hello,\r\nI have create a database backup using codeigniter $this->load->dbutil(); This Work fine and I can download my database backup. But I try to restore this backup and I got below error..\r\n\r\nSQL query:\r\n\r\n\r\nINSERT INTO `app_user` (`id`, `user_name`, `email`, `password`, `role`, `status`) VALUES (10, Admin.sgh, admin@supreme.lk, 202cb962ac59075b964b07152d234b70, 0, 0)\r\nMySQL said: Documentation\r\n\r\n#1064 - You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '@supreme.lk, 202cb962ac59075b964b07152d234b70, 0, 0)' at line 1\r\n\r\n**Can you help me to resolve this error. I am using codeigniter  3.1.6 Version..** ","score":1.0}]}
0

